src/frontend/web_application/src/index.jsx

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Router } from 'react-router-dom';
 import PiwikReactRouter from 'piwik-react-router';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Where is that dependency ? Local ?

Reply:
actually it's due to the not good enough project directory's structure:

This package is not required to run in production, because it is bundled for the client and the `index.jsx` file is only used by the client build.

It would be different if the client build has it's own `package.json`.